api/client.go

```diff
@@ -18,8 +18,8 @@ import (
 //go:generate mockery --name=httpClient  --structname=HttpClient --output=../mocks/api
 
 const (
-	httpScheme  = "http"
-	httpsScheme = "https"
+	HttpScheme  = "http"
+	HttpsScheme = "https"
```

Comment:
```suggestion
	HTTPScheme  = "http"
	HTTPSScheme = "https"
```

Can you update the constants to follow golang idiomatic casing for initialisms?